.github/workflows/render-and-upload-manifests-reusbale.yml

```diff
@@ -0,0 +1,60 @@
+name: Render and upload manifests (reusable)
```

Comment:
The filename is misspelled:
`render-and-upload-manifests-reusbale.yml` -> `render-and-upload-manifests-reusable.yml`

Reply:
thanks! Resolved.